tests/Cache/CacheMemcachedStoreTest.php

```diff
@@ -63,12 +63,10 @@ public function testSetMethodProperlyCallsMemcache()
             $this->markTestSkipped('Memcached module not installed');
         }
 
-        \Illuminate\Support\Carbon::setTestNow($now = \Illuminate\Support\Carbon::now());
         $memcache = $this->getMockBuilder('Memcached')->setMethods(['set'])->getMock();
-        $memcache->expects($this->once())->method('set')->with($this->equalTo('foo'), $this->equalTo('bar'), $this->equalTo($now->timestamp + 60));
+        $memcache->expects($this->once())->method('set')->with($this->equalTo('foo'), $this->equalTo('bar'), $this->equalTo($this->now->getTimestamp() + 60));
         $store = new MemcachedStore($memcache);
         $store->put('foo', 'bar', 1);
-        \Illuminate\Support\Carbon::setTestNow();
```

Comment:
> This is happening mid test though, so is it not resetting the time. which is likely to be wanted to be kept.

No. it's at the end of the test. It's required as you have ` \Illuminate\Support\Carbon::setTestNow($now = \Illuminate\Support\Carbon::now());` before